assets/css/static-main.css

```diff
@@ -466,13 +546,89 @@ a:active {
     font-size: 18px
 }
 
+.ds-c-link, a {
+    color: #0071bc;
+    cursor: pointer;
+    text-decoration: underline
+}
+
+.ds-c-link:visited, a:visited {
+    color: #4c2c92
+}
+
+.ds-c-link:hover, a:hover {
+    color: #205493
+}
+
+.ds-c-link:focus, .ds-c-link:focus:visited, a:focus, a:focus:visited {
+    background-color: #ffde11;
```

Comment:
Is this a color that is required by 508 Compliance, or one that we selected?

Reply:
@msnook It's the color chosen by the design system.  I've overridden it in the last commit; do you like this better?